pegasus/sites.v3/code.org/public/educate/professional-learning/middle-high.haml

```diff
@@ -23,6 +23,7 @@ theme: responsive_full_width
         %li=hoc_s(:pl_middle_high_top_list_01)
         %li=hoc_s(:pl_middle_high_top_list_02)
         %li=hoc_s(:pl_middle_high_top_list_03)
+        %li=hoc_s("pl_page_k5.top_list_04")
```

Comment:
@dmcavoy added this re: [this Figma comment](https://www.figma.com/file/sZ6d1uhJWFksJ6j3Fnmbnc?type=design&node-id=1429-5376&mode=design#708856675)
<img width="1083" alt="Screenshot 2024-04-10 at 1 08 14 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/9d8e80b1-eae0-4045-af32-1cba61c533f2">